src/Hl7.Fhir.Shims.STU3AndUp/Model/ModelInfo.cs

```diff
@@ -56,6 +56,7 @@ private string DebuggerDisplay
 
             public string? Resource { get; set; }
             public string? Name { get; set; }
+            public string? Code { get; set; }
```

Comment:
`Name` and `Url` were already there and removing the nullable here would break the interface. We could change the `Code` to `string` instead of `string?`.